tools/topology/topology2/include/pipelines/cavs/passthrough-wov.conf

```diff
@@ -0,0 +1,135 @@
+#
+# CAVS passthrough capture pipeline
+#
+# A simple passthrough pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.passthrough-capture.N.attribute_name".
+#
+# Usage: passthrough-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.passthrough-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/micsel.conf>
+<include/components/dummywov.conf>
+<include/components/pipeline.conf>
+<include/controls/bytes.conf>
+<common_definitions.conf>
+
+Class.Pipeline."passthrough-wov" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!mandatory [
+			"format"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# passthrough-capture objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+
+		virtual."virtual.detect_sink" {
+			type out_drv
+		}
+
+		micsel."1" {
+			Object.Base.audio_format.1 {
+				in_rate		16000
+				out_rate	16000
+				in_channels	2
+				out_channels	1
+				in_ch_map	CHANNEL_MAP_STEREO
+				in_ch_cfg	1
+				out_ch_map	CHANNEL_MAP_MONO
+				out_ch_cfg	0
+				out_bit_depth	32
+				in_bit_depth	32
+				in_valid_bit_depth	32
+				out_valid_bit_depth	32
+			}
+		}
+
+		dummywov."1" {
+			Object.Base.audio_format.1 {
+				in_rate		16000
+				out_rate	16000
+				in_channels	1
+				out_channels	1
+				in_ch_cfg	0
+				in_ch_map	CHANNEL_MAP_MONO
+				out_bit_depth	32
+				in_bit_depth	32
+				in_valid_bit_depth	32
+				out_valid_bit_depth	32
+				ibs			0x80
+				obs			0x80
+			}
+			Object.Control.bytes."1" {
+				name wov_init
+				access  [
+					tlv_read
+					tlv_callback
+				]
+				Object.Base.extops."extctl" {
+					get 258
+					put 0
+				}
+				max 4096
+				# TODO: use tool to generate the blob and include the blob file
+				Object.Base.data."InitPayload" {bytes "0x53, 0x4f, 0x46, 0x34, 0x14, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x34, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xff, 0x1f, 0x00, 0x00, 0x34, 0x08, 0x00, 0x00"}
```

Comment:
Can we add a comment to describe the blob's struct?

Reply:
done